LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2019- Slack Technologies, LLC
+Copyright (c) 2022- Slack Technologies, LLC
```

Comment:
This template started this year. Also, I was thinking of a better placeholder than the Slack company here as the app will be owned by external devs... but I could not come up with any great ideas.

Reply:
"Slack Technologies, LLC et al." could be a placeholder to complete the license while still leaving space for the external dev, but I'm not sure if this makes sense in a license..